### PR TITLE
[ODS-4578] Bulk Load Client Errors

### PR DESCRIPTION
**Pull request by jamessmckay**

### CODE REVIEW

File: logistics/scripts/modules/settings/settings-management.psm1 [hunk 1]
```diff
@@ -17,6 +17,7 @@ function Get-DefaultDevelopmentSettingsByProject {
             }
             ConnectionStrings = @{ }
             Plugin            = @{
+                Folder = "../../Plugin"
```
Comment (blmeyers) [inline logistics/scripts/modules/settings/settings-management.psm1:20]: So what should the default Folder be for the appsettings.json?

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: logistics/scripts/modules/settings/settings-management.psm1 (hunk 1 above)
Comment (jamessmckay) [reply logistics/scripts/modules/settings/settings-management.psm1:20]: good question, i need this to make the populated template to work. but I was unsure if I should change production version of the link in the appsettings.json.

Comment (blmeyers) [reply logistics/scripts/modules/settings/settings-management.psm1:20]: This folder is moving anyways into WebApi, so this is fine for now.

**Review by blmeyers — APPROVED** [review]

**Comment by blmeyers** [comment]
Issues with Context usage will be addressed in a future ticket.  Merging in so we can get templates building again.

**Pull request merged**